data/releases/schedule.yaml

```diff
@@ -1,4 +1,10 @@
 schedules:
+- release: 1.25
+  releaseDate: 2022-08-23
+  next: 1.25.1
+  cherryPickDeadline: "TBD"
+  targetDate: "TBD"
+  endOfLifeDate: "TBD"
```

Comment:
These aren't parsing as dates.

Reply:
I guess it is ok to leave it empty as of now. 
And I assume we don't need cherryPickDeadline?

Reply:
initially I left out `next`, `cherryPickDeadline`, `targetDate`, and `endOfLifeDate` and the build failed
Build also failed with a string ("TBD") instead of valid dates

Reply:
Build succeeded with
```
- release: 1.25
  releaseDate: 2022-08-23
  next: 1.25.1
  cherryPickDeadline: 2022-09-09
  targetDate: 2022-09-14
  endOfLifeDate: 2023-10-31
```

Reply:
Looks like we have to have entries for  `next`, `cherryPickDeadline`, `targetDate`, and `endOfLifeDate` with valid dates